wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -96,6 +96,21 @@ public final void addRequirements(Subsystem... requirements) {
     }
   }
 
+  /**
+   * Adds the specified subsystems to the requirements of the command. The scheduler will prevent
+   * two commands that require the same subsystem from being scheduled simultaneously.
+   *
+   * <p>Note that the scheduler determines the requirements of a command when it is scheduled, so
+   * this method should normally be called from the command's constructor.
+   *
+   * @param requirements the requirements to add
+   */
+  public final void addRequirements(Set<Subsystem> requirements) {
```

Comment:
Can this accept a `Collection` or even an `Iterable`? Is there a reason to narrow the parameter type to a `Set`?

Maybe uniqueness, but it's not checked in the vararg overload so I'm not sure how much benefit we get from doing it here.